nh_eobs_mobile/static/dev/coffee/src/nhmobilepatient.coffee

```diff
@@ -129,6 +134,10 @@ class NHMobilePatient extends NHMobile
         chartFunc(self, serverData)
       if validTable
         tableFunc(self, serverData)
+      if activeTab is "#graph-content"
+        tableEl.style.display = "none"
+      else
+        graphContent.style.display = "none"
```

Comment:
I meant we could still just hit the server once, but keep the dict / list / whatever from the server in memory until tab load when we do all the JS operations required to draw the table. I'm not sure how expensive they are, probably not very?